James_Bond/mr_bond.py

```diff
@@ -0,0 +1,46 @@
+def input_func():
+    """Receives an input 
```

Comment:
Docstring should be of the format:
```
"""Summary description of the function.

    <Optional longer description> 

    Args:
        param1 (type): The first parameter or None if no parameters.
        param2 (type): The second parameter.

    Returns:
       (return type) bool: The return value for the function. eg. True for success, False otherwise.
    """
```